packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -302,6 +325,18 @@ class OpPerfTelemetry {
 				opPerfData: {},
 			});
 		}
+
+		if (message.type == MessageType.NoOp) {
+			// Count the number of no-ops submitted by this client.
+			// The value is reset when we log the OpStats sampled event.
+			this.noOpCount++;
+		}
+
+		this.opsLogger.sendPerformanceEvent({
+			eventName: "OpStats",
```

Comment:
This event name is too generic as we use the term "op" for all message types and it can be confusing. Maybe "NoOpStats"?

Reply:
Intention is to keep it generic